charts/open-notificaties/templates/configmap.yaml

```diff
@@ -12,6 +12,8 @@ data:
   DB_NAME: {{ .Values.settings.database.name | toString | quote }}
   CACHE_DEFAULT: {{ .Values.settings.cache.default | toString | quote }}
   CACHE_AXES: {{ .Values.settings.cache.axes | toString | quote }}
+  DEBUG: {{ .Values.settings.debug | toString | quote }}
+  IS_HTTPS: {{ .Values.settings.isHttps | toString | quote }}
```

Comment:
Would be great to sort this alphabetically.

Reply:
I sorted the configmap.yaml and secret.yaml of the open-notificaties and the open-zaak chart.
Notice that I kept some entries in 'logical' order. i,e,:
```
  DB_HOST: {{ .Values.settings.database.host | toString | quote }}
  DB_PORT: {{ .Values.settings.database.port | toString | quote }}
  DB_USER: {{ .Values.settings.database.username | toString | quote }}
  DB_NAME: {{ .Values.settings.database.name | toString | quote }}
```
This seems a logical order to me.